### PR TITLE
Fix within/writable spelling in `.\src\USBHost` and `.\src\USBHost3GModule`

### PR DESCRIPTION
**Pull request by aliphys**
The current status on the Readme page states that the Spelling Check is not passing.

bild

This PR resolves issues in https://github.com/arduino-libraries/Arduino_USBHostMbed5/actions/runs/6708967539 to make sure that the spell check passes https://github.com/arduino-libraries/Arduino_USBHostMbed5/actions/runs/6723001780

### CODE REVIEW

File: src/USBHost3GModule/WANDongleSerialPort.h [hunk 1]
```diff
@@ -81,7 +81,7 @@ class WANDongleSerialPort : public IUSBHostSerial {
     *
     * @returns the number of bytes available
     */
-    virtual int writeable();
```
Comment (aliphys) [inline src/USBHost3GModule/WANDongleSerialPort.h:84]: Hi @cristidragomir97, Just wanted to point out that I changed the `writeable()` methods to `writable()` across the entire repository. This change shouldn't create any problems, should it?

Comment (cristidragomir97) [reply src/USBHost3GModule/WANDongleSerialPort.h:84]: I'm not sure about the implications of this on other libraries/examples. Let's ask @facchinm.

**Comment by aliphys** [comment]
We should bump the version number in the next release, since the function rename could potentially be a breaking change.

File: src/USBHost3GModule/WANDongleSerialPort.h (hunk 1 above)
Comment (cristidragomir97) [reply src/USBHost3GModule/WANDongleSerialPort.h:84]: Looks good to me

**Review by cristidragomir97 — APPROVED** [review]

**Pull request merged**